### ISSUE TITLE
多进程跑出现loss为0，eval loss为nan

### ISSUE THREAD

**Issue by heshuguo** [ymcui/Chinese-LLaMA-Alpaca#319]
感谢您使用Issue提问模板，请按照以下步骤提供相关信息。我们将优先处理信息相对完整的Issue，感谢您的配合。

*提示：将[ ]中填入x，表示打对钩。提问时删除上面这两行。请只保留符合的选项，删掉其他。*

### 详细描述问题
采用多个进程微调chinese_lora_alpaca_plus_13b模型的时候出现loss为0，并且eval loss为nan，padding_side为right

### 运行截图或log

![image](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/19610534/04480a3b-1e09-4a0f-9a2b-43c0be4f7fac)

运行命令如下：
WORLD_SIZE=2 CUDA_VISIBLE_DEVICES=1,2 torchrun --nproc_per_node=2 finetune.py --base_model '/data/public-model/plus-13b-lora/merge_chinese_lora_alpaca_plus_13b' --data_path './data/merge-46w.json' --output_dir "./plus-13b-output/alpaca-plus-13b-test-001" --batch_size 32 --micro_batch_size 16 --num_epochs 2 --learning_rate 3e-4 --cutoff_len 512 --val_set_size 5000 --lora_r 64 --lora_alpha 128 --lora_dropout 0.1 --lora_target_modules '[q_proj,k_proj,v_proj,o_proj,gate_proj,down_proj,up_proj]' --train_on_inputs --group_by_length

这个大佬们抽空能帮忙解答下大概什么问题？

**Comment by heshuguo** [comment]
@airaria @ymcui 能帮忙看看上个问题大概是什么原因吗？7b的时候微调padding_side改成right后loss和eval loss正常，但是13b的时候loss和eval loss全是0

**Comment by ymcui** [comment]
出nan的可能原因太多了，建议你看一下我们最新push的精调代码和超参，对照比对一下你的设置。

**Comment by Antiman-cmyk** [comment]
我也是遇到了训练过程中的loss 和 learning rate始终为0，该如何解决呢？并且loss scale一直下降到了0
[deepspeed] OVERFLOW! Rank 0 Skipping step. Attempted loss scale: 1, reducing to 0
[deepspeed] OVERFLOW! Rank 0 Skipping step. Attempted loss scale: 0, reducing to 0
...
Exception: Current loss scale already at minimum - cannot decrease scale anymore. Exiting run.

**Comment by zhl5842** [comment]
问题解决了么？同+1

**Comment by zzx16045223** [comment]
> 问题解决了么？同+1

请问解决了吗，我也有同样的问题
